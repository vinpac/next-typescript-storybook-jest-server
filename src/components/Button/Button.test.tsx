/* eslint-env jest */
// import React from 'react'
// import { create } from 'react-test-renderer'
// import Button from '.'

describe('NiceCheckbox', () => {
  it('renders the checkbox with correct label', () => {
    // expect(create(<Button color="blue" />).toJSON()).toMatchSnapshot()
    // expect(create(<Button />).toJSON()).toMatchSnapshot()
  })
})
