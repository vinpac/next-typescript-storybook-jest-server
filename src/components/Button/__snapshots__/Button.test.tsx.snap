// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NiceCheckbox renders the checkbox with correct label 1`] = `
<button
  style={
    Object {
      "backgroundColor": "#fff",
      "color": "#eee",
      "fontSize": "2rem",
      "padding": 40,
    }
  }
>
  <div
    className="btn btn-primary"
  >
    Home
  </div>
</button>
`;

exports[`NiceCheckbox renders the checkbox with correct label 2`] = `
<button
  style={
    Object {
      "backgroundColor": "#fff",
      "color": "#eee",
      "fontSize": "2rem",
      "padding": 40,
    }
  }
>
  <div
    className="btn btn-primary"
  >
    Home
  </div>
</button>
`;
